### DISCUSSION TITLE
Looking for a security contact to disclose vulnerabilities

### DISCUSSION THREAD

**Discussion by paul-gerste-sonarsource** [jackc/pgx#1910] [Q&A] [answered]
Hi! 👋 

I found and verified two security vulnerabilities affecting the latest pgx v4 and v5 versions.

I want to disclose those issues to the maintainers responsibly, so I'm looking for a security contact (e.g., an email address) to send the details privately.

Thanks!

**Comment by jackc** [comment, marked as answer]
You can reach me (@jackc) at jack@jackchristensen.com.
